README.md

```diff
@@ -0,0 +1,52 @@
+## How to build
+1. Fork and clone the repositry
+2. Open the project folder and open a terminal within the project's directory.
+3. Use the command `premake5 vs2022`
+   - You can change the files generated by using a different [action](https://premake.github.io/docs/using-premake) than `vs2022`.   
+5. Open the new `Build` folder and double click the `AssetConverter.sln` file.
+6. Set the build configuration to `Release`.
+7. Use `ctrl`+`shift`+`b` to build the solution or right click the solution and select `Build Solution`
+8. The solution builds to `../Build/Bin/AssetConverter/[BuildConfig]`
+
+## How to use
+1. Download the `community-listfile.csv` from [here](https://github.com/wowdev/wow-listfile) and rename it to `listfile`.
+2. Go to your `../World of Warcraft/_classic"` folder and copy the `listfile.csv` and `AssetConverter.exe` to this folder.
+3. Run the `AssetConverter.exe`. If this is the first time running it, close the terminal.
+4. A new file named `AssetConverterConfig` has been created. Open it.
+    - If this is not your first time running the Asset Converter and you don't want to extract certain files, set `"Enabled:"` to `false` to the relavant value(s).
+    - If this is your first time, make sure your file looks like this:
+  ```json
+  {
+    "General": {
+        "Version": "0.4",
+        "ThreadCount": 19,
+        "DebugMode": false
+    },
+    "Casc": {
+        "Locale": "enGB",
+        "ListFile": "listfile.csv"
+    },
+    "Extraction": {
+        "Enabled": true,
+        "ClientDB": {
+            "Enabled": true
+        },
+        "Map": {
+            "Enabled": true,
+            "BlendMaps": true
+        },
+        "MapObject": {
+            "Enabled": true
+        },
+        "ComplexModel": {
+            "Enabled": true
+        },
+        "Texture": {
+            "Enabled": true
+        }
+    }
+   ```
+
+5. Run the `AssetConverter.exe` again. This process may take up to 5 minutes, but varies from setup to setup.
+    - Compiling the project in debug instead of release causes this step to take an exponentionally longer amount of time.
+6. When the Asset Converter is finished, a new file called `Data` will be generated. This file will be used by the [Game](https://github.com/novusengine/Game) and will be discused in that project's README text file.
```

Comment:
"a new file called 'Data'" should probably refer to it as a folder

Reply:
Fixed